Author: Daniel Baumann <mail@daniel-baumann.ch>
Description: Temporarily build without stdarg.h in efistdarg.h (Closes: #747158).

diff -Naurp gnu-efi.orig/inc/efistdarg.h gnu-efi/inc/efistdarg.h
--- gnu-efi.orig/inc/efistdarg.h
+++ gnu-efi/inc/efistdarg.h
@@ -18,9 +18,6 @@ Abstract:
 Revision History
 
 --*/
-#ifdef __GNUC__
-#include "stdarg.h"
-#else
 #define _INTSIZEOF(n)   ( (sizeof(n) + sizeof(UINTN) - 1) & ~(sizeof(UINTN) - 1) )
 
 typedef CHAR8 * va_list;
@@ -29,5 +26,3 @@ typedef CHAR8 * va_list;
 #define va_arg(ap,t)    ( *(t *)((ap += _INTSIZEOF(t)) - _INTSIZEOF(t)) )
 #define va_end(ap)  ( ap = (va_list)0 )
 #endif
-
-#endif  /* _INC_STDARG */
